src/objectExplorer/objectExplorerFilter.ts

```diff
@@ -0,0 +1,87 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ReactWebViewPanelController } from "../controllers/reactWebviewController";
```

Comment:
We're inconsistently casing `WebView` and `Webview` in various identifier and filenames.  Would be great to be consistent there.  Looks like VS Code doesn't capitalize the `v` so I'd recommend going that direction.